.github/CHANGELOG.md

```diff
@@ -311,9 +311,7 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 <h3>Improvements</h3>
 
 
-* The adjoint jacobian differentiation method reuses the state computed on the forward pass. For torch and tensorflow
-interfaces, this is disabled by default. By passing `cache_state=True` during QNode creation for tensorflow and 
-torch interfaces, the user can save the device state for the backward pass.
+* The adjoint jacobian differentiation method reuses the state computed on the forward pass.
```

Comment:
```suggestion
* The adjoint jacobian differentiation method reuses the state computed on the forward pass.
  This can be turned off to save memory with the Torch and TensorFlow interfaces by passing
  `adjoint_cache=False` during QNode creation.
```
Worth mentioning?